ppr-ui/tests/unit/utils/helper-functions.ts

```diff
@@ -120,3 +130,9 @@ export function setupMockStaffUser (): void {
   setupMockUser()
   store.setAuthRoles(['staff', 'ppr_staff'])
 }
+
+export function setupMockQualifiedSupplier (): void {
+  setupMockUser()
+  store.setAuthRoles([])
```

Comment:
Probably need to specify that this is Lawyer QS.
In my QS tests, I did add `[AuthRoles.MHR_TRANSFER_SALE]` roles, with a `[ProductCode.MANUFACTURER]` product codes.

Reply:
Sure, can update. 
Although the Roles no longer have a part to play, for the UI at least. We should just be relying on the product code.